db/seeds.d/020-permissions_list.rb

```diff
@@ -186,6 +186,10 @@ def permissions
         ['User', 'edit_users'],
         ['User', 'destroy_users'],
         [nil, 'view_statuses'],
+        ['TablePreference', 'view_table_preferences'],
+        ['TablePreference', 'create_table_preferences'],
+        ['TablePreference', 'edit_table_preferences'],
+        ['TablePreference', 'destroy_table_preferences'],
```

Comment:
I guess these are leftovers, we are not really using htem now, right?

Reply:
Yes. Thanks!